__tests__/app.test.js

```diff
@@ -39,8 +42,44 @@ describe("GET /api", () => {
       .then(({ body }) => {
         const endpoints = body.endpoints;
 
-        expect(typeof endpoints).toBe("object");
-        expect(endpoints).not.toBeNull();
+        expect(endpoints).toEqual(endpointsData);
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET:200 sends a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
```

Comment:
For your 200 test - to allow for later extension of the endpoint, the test should use a flexible matcher eg expect.objectContaining or .toMatchObject. We only need to specifically check the value of the article_id (to ensure it is the correct article)